code/game/objects/items/decorations.dm

```diff
@@ -214,14 +214,18 @@
 //Decorative structures
 ///////
 
-
 /obj/structure/decorative_structures
 	icon = 'icons/obj/decorations.dmi'
 	icon_state = ""
 	density = TRUE
 	anchored = FALSE
 	max_integrity = 100
 
+/obj/structure/decorative_structures/attackby(obj/item/W, mob/living/user, params)
```

Comment:
You'd want to override `wrench_act` instead. Should be plenty of examples which would help you here

Reply:
Done! ^^ Cheers for the help.